flake.nix

```diff
@@ -18,67 +18,68 @@
         inherit system;
         config.allowUnfree = true;
       };
+      inherit (pkgs) lib;
+
       gdk = pkgs.google-cloud-sdk.withExtraComponents (with pkgs.google-cloud-sdk.components; [
         gke-gcloud-auth-plugin
       ]);
       python = pkgs.python313;
-      pythonPackages = pkgs.python313Packages;
-      envWithScript = script:
-        (pkgs.buildFHSEnv {
-          name = "2i2c-env";
-          targetPkgs = pkgs:
-            [
-              python
-              pkgs.pythonManylinuxPackages.manylinux2014Package
-            ]
-            ++ (with pythonPackages; [pip virtualenv])
-            ++ (with pkgs; [
-              cmake
-              ninja
-              gcc
-              pre-commit
-              # Infra packages
-              go-jsonnet
-              kubernetes-helm
-              kubectl
-              sops
-              gdk
-              awscli2
-              azure-cli
-              terraform
-              eksctl
-            ]);
-          # If there is a higher power, why does it allow such horrible autoformatting below
-          runScript = "${pkgs.writeShellScriptBin "runScript" (''
-              set -e
+      packages =
+        [
+          python
```

Comment:
We don't actually use `virtualenv` (only `venv`), and `pip` is default bundled with `venv`!